packages/openapi-tanstack-query/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "openapi-tanstack-query",
+  "description": "Fast, type-safe fetch client for your OpenAPI schema. Only 5 kb (min). Works with React, Vue, Svelte, or vanilla JS.",
+  "version": "0.0.1",
+  "author": {
+    "name": "Drew Powers",
+    "email": "drew@pow.rs"
+  },
+  "license": "MIT",
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
+      }
+    },
+    "./*": "./*"
+  },
+  "homepage": "https://openapi-ts.dev",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openapi-ts/openapi-typescript",
+    "directory": "packages/openapi-fetch"
+  },
+  "bugs": {
+    "url": "https://github.com/openapi-ts/openapi-typescript/issues"
+  },
+  "keywords": [
+    "openapi",
+    "swagger",
+    "rest",
+    "api",
+    "oapi_3",
+    "oapi_3_1",
+    "typescript",
+    "fetch",
+    "react",
+    "vue",
+    "svelte"
+  ],
+  "scripts": {
+    "build": "pnpm run build:clean && pnpm run build:js && pnpm run build:js-min && pnpm run build:cjs",
+    "build:clean": "del-cli dist",
+    "build:js": "mkdir -p dist && cp src/* dist",
+    "build:js-min": "esbuild --bundle src/index.js --format=esm --minify --outfile=dist/index.min.js && cp dist/index.d.ts dist/index.min.d.ts",
+    "build:cjs": "esbuild --bundle src/index.js --format=cjs --outfile=dist/cjs/index.cjs && cp dist/index.d.ts dist/cjs/index.d.cts",
+    "format": "biome format . --write",
+    "lint": "biome check .",
+    "generate-types": "openapi-typescript test/fixtures/api.yaml -o test/fixtures/api.d.ts",
+    "pretest": "pnpm run generate-types",
+    "test": "pnpm run \"/^test:/\"",
+    "test:js": "vitest run",
+    "test:ts": "tsc --noEmit",
+    "version": "pnpm run prepare && pnpm run build"
+  },
+  "dependencies": {
+    "@tanstack/react-query": "^5.45.1",
```

Comment:
This would be better as a `"peerDependency"` instead (with a looser range, e.g. `"5.x"`). We don’t want to install this for users who have this in their project already; it could lead to version mismatches (or type mismatches)

```suggestion
```

Reply:
Yes definitely, same for `openapi-fetch`